addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -24,37 +30,62 @@ const FacebookPageWidget = publicWidget.Widget.extend({
             params.href = `https://www.facebook.com/${params.id}`;
         }
         delete params.id;
-        params.width = utils.confine(Math.floor(this.$el.width()), 180, 500);
 
-        var src = $.param.querystring('https://www.facebook.com/plugins/page.php', params);
-        this.$iframe = $('<iframe/>', {
-            src: src,
-            width: params.width,
-            height: params.height,
-            css: {
-                border: 'none',
-                overflow: 'hidden',
-            },
-            scrolling: 'no',
-            frameborder: '0',
-            allowTransparency: 'true',
-        });
-        this.$el.append(this.$iframe);
+        this._renderIframe(params);
+
+        this.resizeObserver = new ResizeObserver(debounce(this._renderIframe.bind(this, params), 100));
+        this.resizeObserver.observe(document.querySelector("#wrapwrap"));
```

Comment:
The `wrapwrap` is not the right element to observe.
See here if you modify the width of a column, it doesn't work.
![image](https://github.com/odoo/odoo/assets/52911687/f3ae0fcf-fecc-4b98-a84e-d8eee6ea0ce9)